src/realm/group.hpp

```diff
@@ -510,6 +510,16 @@ class Group : private Table::Parent {
         return !(*this == g);
     }
 
+    /// Compute the sum of the sizes in number of bytes of all the array nodes
+    /// that currently make up this group. When this group represents a snapshot
+    /// in a Realm file (such as during a read transaction via a SharedGroup
+    /// instance), this function computes the footprint of that snapshot withing
+    /// the Realm file.
+    ///
+    /// If this group accessor is the detached state, this function returns
+    /// zero.
+    std::size_t compute_aggregated_byte_size() const noexcept;
```

Comment:
Sure. `std::size_t` is the standard way for header files, but it goes against cores dominant style, so I'll change it.